Review /samples/cosmosdb/transactions (#8339)
Particular/docs.particular.net: samples/cosmosdb/transactions/sample.md

Reviewer: ```suggestion
## Prerequisites

Ensure that an instance of the latest [Azure Cosmos DB Emulator](https://learn.microsoft.com/en-us/azure/cosmos-db/local-emulator) is running.

## Projects
```

Author: The sample only uses this if they haven't configured an environment variable, and I linked to the emulator setup in the config section on line 34.

Reviewer 2: ```suggestion
Most messages implement `IProvideOrderId` and thus it is possible to use the provided order identification (OrderId) as a partition key.
```

### Persistence config

Configure the endpoint to use Cosmos DB Persistence.  The sample is configured to use the [Docker CosmosDB emulator](https://learn.microsoft.com/en-us/azure/cosmos-db/emulator-linux?context=/azure/cosmos-db/context/context#installation) if a connection string is not found in the `COSMOS_CONNECTION_STRING` environment variable.

snippet: CosmosDBConfig

> Because the order id is used as a partition key, it has to be used in the partition key path as well.

Reviewer 2: ```suggestion
Because OrderId is used as a partition key, it has to be used in the partition key path as well.
```

> Because the order id is used as a partition key, it has to be used in the partition key path as well.

Reviewer 2: Question: After staring at this sentence for too long, I don't understand what is trying to point out.
Is it about this line on the snippet `persistence.DefaultContainer("Server", "/OrderId");`? or is it about the section below where it talks about the transaction?

Author: I think it's to clarify the `persistence.DefaultContainer("Server", "/OrderId");`

Good catch, I think it belongs above the snippet and should probably be a little clearer. I'll update.
